### PR TITLE
Matlab mpi

### PR DESCRIPTION
**Pull request by desmfryan**
Matlab multi-node mpi on ShARC documentation.

### CODE REVIEW

**Comment by willfurnass** [comment]
@desmfryan Can we add info on what you/Mathworks did to get this working properly?

File: sharc/software/apps/matlab.rst [hunk 1]
```diff
@@ -239,5 +240,9 @@ In more detail:
...
     chmod -R g+w ${USER}:app-admins /usr/local/packages/apps/matlab/R2017b /usr/local/modulefiles/apps/matlab/2017b
 
-The modulefile is
-:download:`/usr/local/modulefiles/apps/matlab/2017b/binary </sharc/software/modulefiles/apps/matlab/2017b/binary>`.
+#. The modulefile is :download:`/usr/local/modulefiles/apps/matlab/2017b/binary </sharc/software/modulefiles/apps/matlab/2017b/binary>`.
+
+#. Copy integration scripts to MATLAB local directory (required for MATLAB parallel (multi-node)): ::
+
+    cd /usr/local/packages/apps/matlab/2017b/binary/toolbox/local
+    cp -r /usr/local/packages/apps/matlab/parallel_mpi_integration_scripts/* .
```
Comment (willfurnass) [inline sharc/software/apps/matlab.rst]: @desmfryan Are these scripts version-controlled?  If they are in an existing repo could you add a link to that here?  If not, (and there's nothing confidential in them) could they at least be added to this repo?

Comment (desmfryan) [reply sharc/software/apps/matlab.rst]: Hi Will I am sure that the said scripts are version controlled by Mathworks but we cannot link to these...the scripts were provided ad-hoc by Sam Marshalik (by email), and has advised me that they will work for Matlab 2018....due to be installed by me soon.

There are 4 integration scripts plus 2 folders containing additional support scripts. I could create a tar.gz file (of these) with a link in the documentation to it, however where exactly do you think these should be stored (.sheffield_hpc/sharc/software/apps/matlab/). You mean the doc repo right or am I missing something?

Cheers,

Des

Comment (desmfryan) [reply sharc/software/apps/matlab.rst]: @willfurnass see above comment...thanks.

**Pull request merged**